x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -290,3 +306,90 @@ export const importRulesWithOverwriteAll = (rulesFile: string) => {
   cy.get(RULE_IMPORT_MODAL_BUTTON).last().click({ force: true });
   cy.get(INPUT_FILE).should('not.exist');
 };
+
+export const clickAddIndexPatternsMenuItem = () => {
+  cy.get(BULK_ACTIONS_BTN).click();
+  cy.get(INDEX_PATTERNS_RULE_BULK_MENU_ITEM).click();
+  cy.get(ADD_INDEX_PATTERNS_RULE_BULK_MENU_ITEM).click();
+};
+
+export const openBulkEditAddIndexPatternsForm = () => {
+  clickAddIndexPatternsMenuItem();
+
+  cy.get(RULES_BULK_EDIT_FORM_TITLE).should('have.text', 'Add index patterns');
+};
+
```

Comment:
Same as with screens, I'd suggest extracting newly added tasks related to the rules table to `security_solution/cypress/tasks/rules/rules_table.ts`